src/app/components/Footer/index.tsx

```diff
@@ -63,6 +64,15 @@ export function Footer() {
             </p>
           </div>
         </div>
+        <div className="text-center small text-lightGrey">
+          <a
+            href={`https://github.com/DistributedCollective/Sovryn-frontend/commit/${commitHash}`}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {commitHash}
```

Comment:
Your suggestion in issue is valid, lets keep it short like github does:
```suggestion
            {commitHash.substr(0, 7)}
```

Reply:
Updated now